hl7.py

```
@@ -1,31 +1,169 @@
 from datetime import datetime
 from pathlib import Path
 
-from fastapi import FastAPI
+
+import FastAPI
 import asyncio
 
-app = FastAPI()
+from hl7apy.core import Message
+from hl7apy.parser import parse_message
+from hl7apy.core import Message
```

Ruff (0.12.2)

Redefinition of unused `Message` from line 8

Remove definition: `Message`

(F811)